Documentation/ExtensionArchitecture/FileStructure/ComposerJson.rst

```diff
@@ -90,6 +90,11 @@ Subsequently:
 Extended composer.json
 ======================
 
+..  note::
+
+    Please see the chapter on :ref:`section on testing <testing-extensions>` for
```

Comment:
" ... the chapter on section on testing ..." sounds strange..., maybe: "... the section of testing ..." is enough?

Reply:
done